eli5/sklearn/transform.py

```diff
@@ -21,15 +18,23 @@
 
 # Feature selection:
 
-@transform_feature_names.register(SelectorMixin)
-@transform_feature_names.register(RandomizedLogisticRegression)
-@transform_feature_names.register(RandomizedLasso)
 def _select_names(est, in_names=None):
     mask = est.get_support(indices=False)
     in_names = _get_feature_names(est, feature_names=in_names,
                                   num_features=len(mask))
     return [in_names[i] for i in np.flatnonzero(mask)]
 
+try:
+    from sklearn.linear_model import (  # type: ignore
+        RandomizedLogisticRegression,
+        RandomizedLasso,
+    )
+    _select_names = transform_feature_names.register(RandomizedLasso)(_select_names)
+    _select_names = transform_feature_names.register(RandomizedLogisticRegression)(_select_names)
+except ImportError:     # Removed in scikit-learn 0.21
+    pass
+_select_names = transform_feature_names.register(SelectorMixin)(_select_names)
```

Comment:
``SelectorMixin`` could be left as a decorator, right?

Reply:
You are correct. I expanded out and placed `SelectorMixin` last to keep the original order that the decorators are applied in (RandomizedLasso -> RandomizedLogisticRegression -> SelectorMixin). But in this case order does not seem to matter.